src/main/java/io/github/hapjava/server/impl/json/AccessoryController.java

```diff
@@ -58,8 +59,9 @@ public HttpResponse listing() throws Exception {
     }
 
     try (ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
-      Json.createWriter(baos)
-          .write(Json.createObjectBuilder().add("accessories", accessories).build());
+      JsonWriter writer = Json.createWriter(baos);
+      writer.write(Json.createObjectBuilder().add("accessories", accessories).build());
+      writer.close();
```

Comment:
JsonWriter is an AutoClosable, makes more sense to use the try block:

try {(ByteArrayOutputStream baos = new ByteArrayOutputStream();
        JsonWriter writer = Json.createWriter(baos))